controllers/storage/minio.go

```diff
@@ -50,7 +50,19 @@ func (m *MinIOReconciler) Reconcile() (*lcm.CRStatus, error) {
 	var minioCR minio.MinIOInstance
 
 	if m.HarborCluster.Spec.Storage.Kind != inClusterStorage {
-		return m.ProvisionExternalStorage()
+		var exSecret corev1.Secret
+		err := m.KubeClient.Get(m.getExternalSecretNamespacedName(), &exSecret)
```

Comment:
Couple  things I think we should aware of:
1. switch external to inCluster 
1. switch inCluster to external

Reply:
Implements it in the next version.